README.md

```diff
@@ -213,7 +213,7 @@ migrated to use the `--entry.name file` syntax.
 ### Resolve Alias
 
 Resolve aliases passed by flag in `webpack-cli` using
-`--resolve-alias alias=value` should be migrated to use the `--entry.name file`
+`--resolve-alias alias=value` should be migrated to use the `--resolve-alias.alias value`
```

Comment:
let's improve this a little for the users (because it admittedly confused me as I haven't looked at it in a while)

```
should be migrated to use the `--resolve-alias.{alias} {value}`
```

Hopefully that'll give people the indication those are template values

Reply:
makes sense 👍